AsyncDisplayKit/ASLayoutElementInspectorCell.m

```diff
@@ -197,23 +222,23 @@ - (void)buttonTapped:(ASButtonNode *)sender
   BOOL selfIsEditing = (self == __currentlyOpenedCell);
   [__currentlyOpenedCell endEditingValue];
   if (selfIsEditing) {
+    sender.selected = NO;
     return;
   }
   
 //  NSUInteger currentAlignSelfValue;
 //  NSUInteger nextAlignSelfValue;
 //  CGFloat    newValue;
-  
+
+  sender.selected = !sender.selected;
```

Comment:
The toggle existed in descender before, but the behavior was inconsistent:
- Select descender [descender button state: selected]
- Select flexGrow [descender button state: selected]
- Select descender [descender button state: deselected]

Based on the initial code, I figured that the goal was to toggle the option on while it's being edited (and then off when editing is done), as well as keep it on for boolean values that are "YES".

The code that I pushed maintains a more consistent button state for that type of behavior.